runtime/oti/j9javaaccessflags.h

```diff
@@ -34,7 +34,7 @@
 #define J9AccAbstract 0x400
 #define J9AccAnnotation 0x2000
 #define J9AccBridge 0x40
-#define J9AccClassAnnnotionRefersDoubleSlotEntry 0x80000
+#define J9AccClassAnnotationRefersDoubleSlotEntry 0x80000
```

Comment:
Please don't change this one, at least for now. This affects DDR and backward compatibility for reading older core files.

Reply:
or maybe not a problem since it isn't used in the DDR code. @keithc-ca any opinion?

Reply:
Constant in general can't be renamed (even if they're mis-spelled) without implications for DDR.

The problem is that the old names will be used in customers' core files which we want to continue to be able to analyse. Changing the name will prevent it a use being introduced to DDR code without accepting that old core files will be misunderstood.